sql/core/src/test/resources/sql-tests/inputs/udf/pgSQL/udf-select_having.sql

```diff
@@ -0,0 +1,57 @@
+--
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- This test file was converted from inputs/pgSQL/select_having.sql
+--
+-- SELECT_HAVING
+-- https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/sql/select_having.sql
+--
```

Comment:
I would comment this:

```
--
-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
--
--
-- SELECT_HAVING
-- https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/sql/select_having.sql
--
-- This test file was converted from inputs/pgSQL/select_having.sql
```

to keep the original comment intact.

Reply:
Corrected